### PR TITLE
fixed the typo errors

### PR DESCRIPTION
**Pull request by jitendragangwar123**
I have fixed the typo errors. Please review and merge it

### CODE REVIEW

File: Lottery-Game-Chainlink-VRF.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ So we can use some web2 technologies to generate the randomness and then use the
 ## What is an oracle?
 
 - An oracle sends data from the outside world to a blockchain's smart contract and vice-versa.
-- Smart contract can then use this data to make a decision and change its state.
+- Smart contract can then use this data to make a decision and change it's state.
```
Comment (yanukadeneth99) [inline Lottery-Game-Chainlink-VRF.md]: This should be `its`. You can find more [here](https://www.grammarly.com/blog/its-vs-its/).

File: Lottery-Game-Chainlink-VRF.md [hunk 2]
```diff
@@ -11,13 +11,13 @@ So we can use some web2 technologies to generate the randomness and then use the
...
 - They act as bridges between blockchains and the external world.
-- However it is important to note that the blockchain oracle is not itself the data source but its job is to query, verify and authenticate the outside data and then further pass it to the smart contract.
+- However it is important to note that the blockchain oracle is not itself the data source but it's job is to query, verify and authenticate the outside data and then further pass it to the smart contract.
```
Comment (yanukadeneth99) [inline Lottery-Game-Chainlink-VRF.md]: This should be `its`. You can find more [here](https://www.grammarly.com/blog/its-vs-its/).

File: Lottery-Game-Chainlink-VRF.md [hunk 3]
```diff
@@ -226,9 +226,9 @@ contract RandomWinnerGame is VRFConsumerBase, Ownable {
 The constructor takes in the following params:
 
 - `vrfCoordinator` which is the address of the VRFCoordinator contract
-- `linkToken` is the address of the link token which is the token in which the chainlink takes its payment
+- `linkToken` is the address of the link token which is the token in which the chainlink takes it's payment
```
Comment (yanukadeneth99) [inline Lottery-Game-Chainlink-VRF.md]: This should be `its`. You can find more [here](https://www.grammarly.com/blog/its-vs-its/).

File: Lottery-Game-Chainlink-VRF.md [hunk 4]
```diff
@@ -318,7 +318,7 @@ This function first checks if our contract has Link token before we request for
     }
 ```
 
-This function was inherited from `VRFConsumerBase`. It is called by `VRFCoordinator` contract after it recieves the randomness from the external world. After recieving the randomness which can be any number in the range of uint256 we decrease its range from `0 to players.length-1` using the mod operaator
+This function was inherited from `VRFConsumerBase`. It is called by `VRFCoordinator` contract after it receives the randomness from the external world. After receiving the randomness which can be any number in the range of uint256 we decrease it's range from `0 to players.length-1` using the mod operator.
```
Comment (yanukadeneth99) [inline Lottery-Game-Chainlink-VRF.md]: This should be `its`. You can find more [here](https://www.grammarly.com/blog/its-vs-its/).

**Review by yanukadeneth99 — CHANGES_REQUESTED** [review]

**Comment by jitendragangwar123** [comment]
I have updated the file according to the given suggestions. Please review and merge it.

**Review by yanukadeneth99 — APPROVED** [review]

**Comment by yanukadeneth99** [comment]
These are great changes. Good stuff @jitendragangwar123! 🔥

**Pull request merged**